### PR TITLE
fix workspace path; fix detail on debian install path

### PR DESCRIPTION
**Pull request by marinagmoreira**

### CODE REVIEW

**Review by bcoltin — COMMENTED** [review]
Can you change the part where it mentions -b on line 82 too

**Comment by oleg-alexandrov** [comment]
Looks good. Once all occurrences are changed it should be good to go.

File: doc/general_documentation/INSTALL.md [hunk 1]
```diff
@@ -74,16 +74,20 @@ by running 'sudo apt-get update' and then 'sudo apt-get upgrade'*
 ## Configuring the build
 
 ### Note for the build setup
-By default, the configure script uses the following paths:
+
```
Comment (marinagmoreira) [inline doc/general_documentation/INSTALL.md:77]: I tried to add a bit of intro for the user to understand a bit of how a workspace works, is it clear?

**Review by bcoltin — APPROVED** [review]
Looks good, thanks!

**Pull request merged**